lib/glific/partners/billing.ex

```diff
@@ -682,6 +681,54 @@ defmodule Glific.Partners.Billing do
     |> Enum.each(&update_period_usage(&1, end_date))
   end
 
+  @doc """
+  Updating monthly usage of organization
+  """
+  @spec update_monthly_usage :: :ok
+  def update_monthly_usage do
```

Comment:
I think we should do that based on active organizations not just the billing

Reply:
@pankaj-ag we are updating the subscription if the organization has an active stripe billing account. if an organization is inactive it will not be having an active billing account